theano/tensor/tests/test_opt.py

```diff
@@ -1811,6 +1815,51 @@ def test_local_useless_subtensor():
         f([[0, 1, 2], [3, 4, 5]])  # let debugmode test something
 
 
+def test_local_subtensor_remove_broadcastable_index():
+    x = T.dmatrix('x')
```

Comment:
Please add a brief description of the test (Comment each line of the description using `#`, so that the name of the test is shown in case of errors)